k8s/object.bzl

```diff
@@ -399,24 +399,26 @@ def k8s_object(name, **kwargs):
   kwargs["image_targets"] = _deduplicate(kwargs.get("images", {}).values())
   kwargs["image_target_strings"] = _deduplicate(kwargs.get("images", {}).values())
 
+  tags = kwargs.get("tags", None)
```

Comment:
From https://docs.bazel.build/versions/master/skylark/lib/globals.html#rule,
> Attributes `visibility`, `deprecation`, `tags`, `testonly`, and `features` are implicitly added and cannot be overridden.

Maybe we should have an `implicit_attrs` dict listing all of these tags, and pass that as kwargs to the instantiated rules?